tests/lib/ecma-features.js

```diff
@@ -34,22 +34,20 @@ const testFiles = shelljs
 // Tests
 //------------------------------------------------------------------------------
 
-describe('ecmaFeatures', () => {
+describe('ecma-features', () => {
```

Comment:
With the removal of `ecmaFeatures`, I felt like it was confusing to not have this match the directory name. As a thought - is there a reason to have `basic` and `ecma-features`? Could it just be one directory called `javascript` or ecmascript` instead?

Reply:
No particulary reason, no, been like that since the beginning. Feel free to change it if you want to, the diff size will skyrocket :D

Reply:
I'd like to do this, but I'll save it for a separate PR :+1: